build-aux/pkg.nix

```diff
@@ -79,19 +79,18 @@ stdenv.mkDerivation (finalAttrs: {
     sed -i -e 's/tarball-version/flake-version/' configure.ac
   '';
 
-  buildInputs =
-    [
-      finalAttrs.finalPackage.passthru.luaEnv
-      cargo-edit
-      harfbuzz
-      icu
-      fontconfig
-      libiconv
-      libxslt
-      stylua
-      taplo
-      typos
-    ];
+  buildInputs = [
+    finalAttrs.finalPackage.passthru.luaEnv
+    cargo-edit
+    harfbuzz
+    icu
+    fontconfig
+    libiconv
+    libxslt
+    stylua
+    taplo
+    typos
+  ];
```

Comment:
This is one of my painful points in the new nix formatting, but I guess I'll have to live with it :).

Reply:
My beef is it did not apply the same style to all elements, e.g. `luaPackages` and `configureFlags` did not get formatted the same way `buildInputs` and others did. :shrug: